src/app/components/room/action.service.js

```diff
@@ -131,5 +132,12 @@
         feed.setEnabledChannel(type, true);
       }
     }
+
+    function setMainFeedChannel(type, boolval) {
```

Comment:
Pretty non-intuitive name for this function. I know that `setEnabledMainFeedChannel` is ridiculous long, but I'm sure we can find a better alternative.